src/Application/Configuration.php

```diff
@@ -35,13 +37,43 @@ class Configuration
      */
     private $source;
 
-    public function __construct(string $key, string $username, string $endpoint, string $version = self::VERSION, ?string $source = self::SOURCE)
+    /**
+     * @var string|null
+     */
+    private $userId;
+
+    /**
+     * @var string|null
+     */
+    private $language;
+
+    /**
+     * @var string|null
+     */
+    private $languageVersion;
+
+    /**
+     * @var string|null
+     */
+    private $integrator;
+
+    /**
+     * @var string|null
+     */
+    private $country;
+
+    public function __construct(string $key, string $username, string $endpoint, string $version = self::VERSION, ?string $source = self::SOURCE, ?string $userId = null, ?string $language = self::LANGUAGE, ?string $languageVersion = null, ?string $integrator = null, ?string $country = null)
```

Comment:
Could you write each parameter in a separated line? This line has almost 300 characters and it makes it poorly legible